src/mtype.d

```diff
@@ -3678,17 +3675,17 @@ public:
             case Tcomplex32:
             case Timaginary32:
             case Tfloat32:
-                fvalue = FLT_EPSILON;
+                fvalue = real_t(FLT_EPSILON);
```

Comment:
Never (even GCC finally dropped ``-fshort-double`` at long last!).  However the problem is converting native -> emulation vs. just asking the emulation layer directly.